device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,100 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '47194112' %}
+{% set ingress_lossless_pool_xoff =  '47194112' %}
```

Comment:
The xoff stands for shared headroom pool size. It should not be equal to pool size.
The buffer pool size is not correct. It's too large for Spectrum 2. You probably used Spectrum 3.
My result is `20664320` xoff `3321856`
- 10G 5m * 2 headroom 44kb
- 50G 5m * 100 headroom 51kb
- 100G 40m * 12 headroom 66kb